bpf_test.go

```diff
@@ -263,4 +291,48 @@ func TestModuleLoadELF(t *testing.T) {
 	checkCgroupProgs(t, b)
 	checkSocketFilters(t, b)
 	checkTracepointProgs(t, b)
+
+	// test 2: New Module, reading from file, with pinning custom config
+	customELF := "./tests/custom.o"
+
+	bc := elf.NewModule(customELF)
+	if bc == nil {
+		t.Fatal("prog is nil")
+	}
+
+	mounted, err := bpffs.IsMounted()
+	if err != nil {
+		t.Fatalf("error checking if %q is mounted: %v", bpffs.BPFFSPath, err)
+	}
+	if !mounted {
+		if err := bpffs.Mount(); err != nil {
+			t.Skipf("error mounting bpf fs, skipping test: %v", err)
+		}
+	}
+
+	var secParams = map[string]elf.SectionParams{
+		"maps/dummy_array_custom": elf.SectionParams{
+			PerfRingBufferPageCount:   0,
+			SkipPerfMapInitialization: false,
```

Comment:
No need to initialize `PerfRingBufferPageCount` and `SkipPerfMapInitialization`: https://play.golang.org/p/88D3orz3xR